content/07.supplemental.md

```diff
@@ -1,24 +1,25 @@
+![**Related to Figure 3.** (A) Sample-specific RefSig signature weights across cancer groups ordered by decreasing Signature 1 exposure. (B) Proportion of Signature 1 plotted by phase of therapy for each cancer group.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/f0ff05e2b0e69d63733030b57088f34737d9f8d4/figures/pngs/figureS4.png?sanitize=true){#fig:S4 tag="S4" width="7in"}
 
-![**Related to Figure 4**. (A) Receiver Operating Characteristic for _TP53_ classifier run on FPKM of poly-A RNA-Seq samples. Correlation plots for telomerase scores (EXTEND) with RNA expression of _TERT_ (B) and _TERC_ (C).First two dimensions from UMAP of sample transcriptome data with points colored by `molecular_subtype` for medulloblastoma (D), ependymoma (E), low-grade glioma (F), and high-grade diffuse astrocytic tumors (G).](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/489d729cde4318654a26b0a9ef0dee6d706352cd/figures/pngs/figureS5.png?sanitize=true){#fig:S5 tag="S5" width="7in"}
+![**Related to Figure 4**. (A) Receiver Operating Characteristic for _TP53_ classifier run on FPKM of poly-A RNA-Seq samples. Correlation plots for telomerase scores (EXTEND) with RNA expression of _TERT_ (B) and _TERC_ (C). First two dimensions from UMAP of sample transcriptome data with points colored by `molecular_subtype` for medulloblastoma (D), ependymoma (E), low-grade glioma (F), and high-grade diffuse astrocytic tumors (G). Box plots of quanTIseq estimates of immune cell proportions in histologies with more than one molecular subtypes with N >=3.](images/supplement/Figure-S5.png){#fig:S5 tag="S5" width="7in"}
 
-**Table S1. Related to Figure 1.** 
+[**Table S1. Related to Figure 1.**](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/tables/results/TableS1-histologies.xlsx)
 Table of specimens and associated metadata, clinical data, and histological data utilized in the OpenPBTA project.
-(Abbreviations: CHDM = chordoma, DMG = diffuse midline glioma, HGG = high-grade glioma, ETMR = embryonal tumor with multilayer rosettes, MB = medulloblastoma, EPN = ependymoma, GNG = ganglioglioma, LGG = low-grade glioma, SEGA = subependymal giant cell astrocytoma, EWS = Ewing sarcoma, CNC = central neurocytoma, EVN = extraventricular neurocytoma, GNT = glial-neuronal tumor, CRANIO = craniopharyngioma)
 
-**Table S2. Related to Figures 2 and 3.** 
+[**Table S2. Related to Figures 2 and 3.**](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/tables/results/TableS2-DNA-results-table.xlsx) 
 Table of TMB, eight CNS mutational signatures, and chromothripsis events per sample.
```

Comment:
```suggestion
Excel file with three sheets representing tables of TMB, eight CNS mutational signatures, and chromothripsis events per sample, respectively.
```

Reply:
I think we should link these somehow in the manuscript - would you prefer elsewhere or was this something you want to link later? I didn't link specific commits for this because they aren't fully merged.